app/actors/hyrax/actors/collections_membership_actor.rb

```diff
@@ -46,6 +52,46 @@ def assign_nested_attributes_for_collection(env, attributes_collection)
               add(env, attributes['id'])
             end
           end
+
+          true
+        end
+        # rubocop:enable Metrics/MethodLength
+        # rubocop:enable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+
+        ##
+        # @deprecated supports old :member_of_collection_ids arguments
+        def emit_deprecation
+          Deprecation.warn(self, ':member_of_collections_attributes and :member_of_collection_ids were both ' \
+                                 ' passed. :member_of_collection_ids is ignored when both are passed and is ' \
+                                 'deprecated for removal in Hyrax 3.0.')
+        end
+
+        ##
+        # @deprecated supports old :member_of_collection_ids arguments
+        def assign_for_collection_ids(env)
+          collection_ids = env.attributes.delete(:member_of_collection_ids)
```

Comment:
The assign_for_collection_ids before the switch to the new style env parameter was...
```
        def assign_collections(env, collection_ids)
          return true unless collection_ids
          multiple_memberships = Hyrax::MultipleMembershipChecker.new(item: env.curation_concern).check(collection_ids: collection_ids)
          if multiple_memberships
            env.curation_concern.errors.add(:collections, multiple_memberships)
            return false
          end
          # grab/save collections this user has no edit access to
          other_collections = collections_without_edit_access(env)
          env.curation_concern.member_of_collections = ::Collection.find(collection_ids)
          env.curation_concern.member_of_collections.concat other_collections
end
```

I don't have a problem with the addition of the exception, but this method needs to include the check for multiple members.